pyroSAR/drivers.py

```diff
@@ -2049,10 +2217,104 @@ def close(self):
         """
         close the database connection
         """
+        self.Session().close()
         self.conn.close()
     
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
+    
+    def drop_element(self, scene, table):
+        """
+        Drop an entry from the database.
+
+        Parameters
+        ----------
+        scene:
+            a SAR scene
+        table: str
+            either data or duplicates
+
+        Returns
+        -------
+        """
+        if table == 'data':
+            self.__drop_element_data(scene)
+        elif table == 'duplicates':
+            self.__drop_element_duplicates(scene)
+        else:
+            print('only data or duplicates can be dropped')
+    
+    def __drop_element_data(self, scene):
```

Comment:
Can we merge this function with the __drop_element_duplicates function? 
They are mostly doing the same except for the different delete statement and they even have the same docstring.